Tests/Form/DataTransformer/ModelsToArrayTransformerTest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Tests\Form\DataTransformer;
+
+use Sonata\AdminBundle\Form\DataTransformer\ModelsToArrayTransformer;
+
+class ModelsToArrayTransformerTest extends \PHPUnit_Framework_TestCase
+{
+    private $modelManager = null;
+    private $choiceList = null;
+
+    public function setUp()
+    {
+        $this->modelManager = $this->getMock('Sonata\AdminBundle\Model\ModelManagerInterface');
```

Comment:
è$this->prophesize('Sonata\AdminBundle\Model\ModelManagerInterface')->getMock()`